Add new method ListProfilesBySubscriptionId and tests

injyzarif opened the pull request:
Based on some swagger changes, the SDK changes include:
- Fixing type from Profile to Endpoint in CreateEndpointAsync method.
- Adding new method ListProfilesbySubscriptionId.
- Adding new tests for the new Profile method.

Commit:
Add new method ListProfilesBySubscriptionId and tests

Based on some swagger changes, the SDK changes include:
- Fixing type from Profile to Endpoint in CreateEndpointAsync method.
- Adding new method ListProfilesbySubscriptionId.
- Adding new tests for the new Profile method.

azurecla commented:
Hi __@injyzarif__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (inzarif). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

stankovski commented:
@azuresdkci retest this please

stankovski commented:
@injyzarif if possible I'd like to wait with this PR until https://github.com/Azure/azure-sdk-for-net/pull/1748 is merged

injyzarif commented:
Do you have an ETA on when this is merged?

stankovski commented:
End of week - when we get all partner teams reply to the request for Client Runtime upgrade

Commit:
Fix SsoUri property and test

- Fixed the SsoUri property name to SsoUriValue.
- Added an assert in the GenerateSsoUri test to check UriValue is
actually not null.

injyzarif commented:
Hi Denis,

Is it possible to merge these before end of week? Kevin, my coworker, is finishing up the PowerShell work and he is stuck on these fixes.

Thanks,
Injy
